interop/cats/src/main/scala/caliban/interop/cats/CatsInterop.scala

```diff
@@ -0,0 +1,31 @@
+package caliban.interop.cats
+
+import caliban.parsing.adt.Value
+import caliban.{GraphQL, ResponseValue}
+import cats.effect.Async
+import zio.{DefaultRuntime, Runtime}
+
+object CatsInterop {
+
+  val defaultZioRuntime = new DefaultRuntime {}
```

Comment:
I think this can be removed, users might already have one and this will create another one. It's even shorter to do  `new DefaultRuntime {}` from the call site 😄

Reply:
I was on the fence about adding it but decided it might be a slightly better UX for someone totally unfamiliar with ZIO, in that everything they need is in the cats interop module and they don't have to look up how to create a runtime (even though it is very simple). 
It's no biggie though so happy to remove it if you think it should go.

Reply:
I'd prefer not having it because it makes it tempting (or just possible by accident) to use 2 different runtimes in different parts of the code, which is a bad practice.

I'm planning to add a documentation page about "how to use caliban with cats effect or monix" once everything is ready, hopefully that will be enough to guide users.